src/rhsmlib/facts/custom.py

```diff
@@ -24,16 +25,16 @@
 
 
 class CustomFacts(object):
-    def __init__(self, data=None):
-        self.data = data
+    def __init__(self, data: Dict[str, Any] = None):
+        self.data: Dict[str, Any] = data
 
     @classmethod
-    def from_json(cls, json_blob):
-        custom_facts = cls
+    def from_json(cls, json_blob: str) -> "CustomFacts":
+        custom_facts: CustomFacts = cls()
```

Comment:
It looks like a bug fix. Shouldn't we deffer this again to another PR. I would add some unit test for this.